core/src/main/scala/cats/arrow/FunctionK.scala

```diff
@@ -124,7 +124,7 @@ private[arrow] object FunctionKMacros extends MacroCompat {
         val G = punchHole(evG.tpe)
 
         q"""
-        new FunctionK[$F, $G] {
+        new _root_.cats.arrow.FunctionK[$F, $G] {
```

Comment:
This is a hygiene issue that I fixed along the way.

Reply:
👍